cfme/infrastructure/host.py

```diff
@@ -544,10 +544,10 @@ def _fill_credential(form, cred, validate=None):
 
 def get_credentials_from_config(credential_config_name):
     creds = conf.credentials[credential_config_name]
-    logger.info('credential name: {cred}'.format(cred=credential_config_name))
-    logger.info('credentials itself: {cred}'.format(cred=creds))
-    return Host.Credential(principal=creds['username'],
-                           secret=creds['password'])
+    logger.info('credential name: %s', credential_config_name)
```

Comment:
@psav, yes that's mine but all the changes made to this file can be removed. @quarckster has found exact reason - #4784.